src/ls/queries.ts

```diff
@@ -77,6 +107,45 @@ ORDER BY
 LIMIT ${p => p.limit || 100};
 `;
 
+const searchTablesAll: IBaseQueries['searchTables'] = queryFactory`
+SELECT U.*, D.database_name AS database
+  FROM (
+      SELECT
+          T.TABLE_NAME AS label,
+          '${ContextValue.TABLE}' AS type,
+          T.TABLE_SCHEMA AS schema,
+          FALSE AS "isView",
+          'table' AS description,
+          ('"' || T.TABLE_SCHEMA || '"."' || T.TABLE_NAME || '"') as detail
```

Comment:
I don't quite understand this line. What is this for?

Reply:
I think they need the fully qualified table name. Since the schema name includes the namespace if appropriate, this will be a name like ns2.s1.t1, or for the non-namespace case, s1.t1.